config.yaml

```diff
@@ -0,0 +1,22 @@
+# OpenID Client configuration
+clients:
+  - id: "client-id" # use this in your oidc client
+    secret: "client-secret" # use this in your oidc client
+    redirect_uris: # URIs of your oidc client
+      [
+        "http://localhost:4000/oidcc/callback",
+        "http://localhost:4000/oidcc/authorize",
+      ]
+
+# Users to log in via sso, and their pls permissions
+users:
+  - ug_kth_id: some-id
+    kth_id: turetek
+    email: turetek@kth.se
+    first_name: Ture
+    family_name: Teknolog
+    pls_permissions:
+      - group: sso
+        permissions: [admin]
+      - group: calypso
+        permissions: [drek, dfunkt]
```

Comment:
Small thing but maybe it would be nicer if this could be defined as:
```yaml
pls_permissions:
  sso:
    - admin
  calypso:
    - drek
    - dfunkt
```
i.e. `Pls_Permissions map[string][]string`